docs/content/community/bijeenkomsten/20181025-demo-sprint-5.md

```diff
@@ -12,8 +12,39 @@ Amsterdam, 25 oktober 2018
 Slides:
 
 * [ZDS2 Demo Spint 5.pdf](/community/bestanden/zds2-demo-sprint-5.pdf)
-* [Opsplitsen RGBZ](/community/bestanden/opsplitsen-rgbz.pdf))
+* [Opsplitsen RGBZ](/community/bestanden/opsplitsen-rgbz.pdf)
 
 ## Feedback
 
-[aanvullen]
+### Wat is de rol en scope van mapper service? Moet dit gestandaardiseerd worden? ###
+todo
+
+### Hoe staat het met authenticatie en autorisatie? ###
+todo
+
+### Hoe gaan jullie om met externe gebruikers m.b.t. authenticatie en autorisatie? ###
+todo
+
+### De URL werkt niet bij update!? ###
+todo
+
+### Wat is er concreet op 20 december en wat is het vervolg? Stip op de horizon? ###
+todo
+
+### Wordt Docker veel gebruikt binnen gemeenten? ###
+todo
+
+### Wat is de huidige toepasbaarheid van de API's? ###
+todo
+
+### Wat was de concrete doelstelling(en) aan het begin van het traject? ###
+Om Zaakgericht Werken een stap verder te brengen worden Zaak- en Documentservices (ZDS) versie 2 ontwikkeld. Hierbij wordt een andere vorm van standaardisatie toegepast. Op basis van relevante informatiemodellen (RGBZ 2.0 en ImZTC 2.2) wordt met zowel publieke als private partijen in een agile proces vorm gegeven aan RESTful API’s die concreet invulling geven aan de gewenste standaard. De standaard wordt tegelijk met een referentie-implementatie ontwikkeld om de implementeerbaarheid aan te tonen, en als referentie te dienen voor latere implementaties.
+
+### Kunnen jullie agenda meesturen in de uitnodiging? ###
+Op de website waar de bijeenkomsten worden aangekondigd en in de definitieve bevestiging wordt een link naar de activiteiten van de huidige sprint opgenomen. https://github.com/VNG-Realisatie/gemma-zaken/projects/3
+
+### Wat verstaan jullie onder backward compatible? ###
+todo
```

Comment:
```suggestion
Grofweg, dat software waarin een API verwerkt is na een update cq. nieuwe versie van de standaard (cq. van één of meer API's) zonder ingrijpende aanpassingen (van die software) kan blijven functioneren. 
```

Reply:
Is een aanzet tot antwoord, niet meer. Feel free ..

Reply:
Definitie prima. Het moet alleen niet de suggestie wekken dat de APIs backwards compatible zijn met StUF-ZKN en/of ZDS 1.x.

> Bij de ontwikkeling van de API’s wordt gestreefd naar backwards compatibility.

Laten we voorop stellen dat leveranciers hoofdzakelijk op RGBZ 1.0 zitten met ZDS 1.1. De overstap naar REST APIs die als uitgangspunt RGBZ 2.0 hebben (waar ook nog aan gesleuteld wordt) wordt gewoon niet triviaal.

Reply:
Helaas klopt die laatste bewering niet: ZDS 1.2 is inhoudelijk gelijk aan ZDS 1.1 maar in de 1.2 versie is een stap gezet naar scherpere berichtschema's. Waar de 1.1 versie nog generieke StUF berichten zoals ZakLk01 en EdcLk01 gebruikte zijn deze in de 1.2 versie vervangen door expliciete berichten als creeerZaak_zakLk01 en updateZaak_zakLk01.

Het plan om RGBZ 2 compliant te zijn middels extraElementen zou in een volgende versie (13.) uitgevoerd worden maar door het besluit in de Regiegroep om deze versie niet meer te ontwikkelen is dat nooit doorgegaan.

Reply:
Joeri schreef:
> Laten we voorop stellen dat leveranciers hoofdzakelijk op RGBZ 1.0 zitten met ZDS 1.1. De overstap naar REST APIs die als uitgangspunt RGBZ 2.0 hebben (waar ook nog aan gesleuteld wordt) wordt gewoon niet triviaal.

Daar ben ik het mee eens, Zowel het onderliggende informatiemodel als de technologie (SOAP vs REST) zijn zodanig anders dat dit een grote stap is.

NB. Afgelopen maandag had ik een gesprek met een grotere leverancier van een zaakserviceconsumer die aangaf dat meerdere leveranciers van zaaksystemen bezig zeggen te zijn met het implementeren van ZDS 1.2. Dit in tegenstelling tot wat deze leveranciers tot eind vorig jaar in de laatste bijeenkomsten van de ZDS werkgroep hebben aangegeven, nl. dat men liever de energie steekt in het inbouwen van ZDS 2.0.  Ik heb geen idee of we iets kunnen met die informatie maar het lijkt er op dat leveranciers al wel behoefte hebben aan een nieuwere, betere standaard.

Reply:
Ik zou inderdaad schetsen dat de huidige situatie is:

* leveranciers/gemeentes zitten op RGBZ 1, en niet 2 & ZDS 1.1
* technologie is wezenlijk verschillend - RESTful API vs. StUF, maar het onderliggend datamodel in implementaties kan waarschijnlijk voor een groot deel hergebruikt worden/op verder gebouwd worden
* we moeten aangeven wat mapt van ZDS 1.1 -> REST (en ZDS 1.2, maar dat gebruiken weinigen blijkbaar)
* we moeten aangeven wat niet te mappen valt

Tegelijkertijd ondersteunen van RESTful APIs & StUF-ZKN kan op database-niveau een compatibiliteitslaag vereisen als iets niet gemapped kan worden. In welke mate dat is, daar kan ik zelf niets over zeggen, ik ben er niet genoeg in thuis. Interessante case hiervoor is wel het zaakmagazijn van Haarlem, daar willen ze toch de REST APIs ook op gaan bouwen?